apollo-api/src/main/java/com/apollographql/apollo/api/Operation.java

```diff
@@ -5,23 +5,54 @@
 
 import javax.annotation.Nonnull;
 
-/** TODO */
+/**
+ * Operation represents an abstraction for a GraphQL operation (mutation or query).
+ */
 public interface Operation<D extends Operation.Data, T, V extends Operation.Variables> {
-  /** TODO */
+  /**
+   * Returns the raw graphQL operation string.
+   */
   String queryDocument();
 
-  /** TODO */
+  /**
+   * Returns the Variable object associated with this GraphQL operation.
+   */
   V variables();
 
+  /**
+   * Returns a mapper that maps the server response back to an object of type D.
```

Comment:
Returns a mapper that maps the server response data  to generated model data class {@link D}